Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "GPL-3.0"
 name = "mdbook-pdf"
 readme = "README.md"
 repository = "https://github.com/HollowMan6/mdbook-pdf"
-version = "0.1.7"
+version = "0.1.8"
```

Comment:
I will make a new release once headless_chrome includes my last PR https://github.com/rust-headless-chrome/rust-headless-chrome/pull/439 so that I can unfork from my dependency https://github.com/HollowMan6/mdbook-pdf/commit/7250bba20eee89f79d5d5d4eea8050d6f4789a79 (should be soon)